wallet-webapps-common/src/main/webapp/vue-app/components/SendTokensForm.vue

```diff
@@ -594,14 +596,20 @@ export default {
       }
       this.$refs.sendTokensForm.open();
       if (!this.isSameNetworkVersion){
-        this.$root.$emit('show-alert', {type: 'warning',message: `${this.$t('exoplatform.wallet.warn.networkVersion')}<br>${this.walletUtils.getNetworkLink()}`});
+        this.showAlert('warning',`${this.$t('exoplatform.wallet.warn.networkVersion')}\n${this.walletUtils.getNetworkLink()}`);
       }
       if (!this.isSameAddress){
```

Comment:
we should add listeners of [Metamask](https://docs.metamask.io/guide/ethereum-provider.html#events) to update the selected network & account of user in order to make this UI more dynamic.